src/hotspot/share/gc/shenandoah/shenandoahGeneration.cpp

```diff
@@ -897,7 +897,7 @@ ShenandoahGeneration::ShenandoahGeneration(ShenandoahGenerationType type,
   _collection_thread_time_s(0.0),
   _affiliated_region_count(0), _humongous_waste(0), _used(0), _bytes_allocated_since_gc_start(0),
   _max_capacity(max_capacity), _soft_max_capacity(soft_max_capacity),
-  _adjusted_capacity(soft_max_capacity), _heuristics(nullptr) {
+  _adjusted_capacity(max_capacity), _heuristics(nullptr) {
```

Comment:
What is `_adjusted_capacity` used for and where? (Given that I see it start out here as being no different from `_max_capacity`, but I'll read on and see if I can find the answer.)

Reply:
Never mind, I see now the code that does adjustments `adjust_available()` & `unadjust_available()`, and IIRC this will be gone (or at least done differently) in Kelvin's upcoming `Expand Old On Demand` work.